scripts/irods/test/test_targeting_specific_replica_number.py

```diff
@@ -0,0 +1,356 @@
+from __future__ import print_function
+import os
+import sys
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from . import session
+from .. import lib
+from .. import test
+from ..test.command import assert_command
+
+class test_targeting_specific_replica_number__issue_6896(
```

Comment:
Are we favoring lowercase instead of uppercase for class names now?
Is there some motivating factor behind this?

Reply:
As near as I can tell, the only reason we use uppercase letters in these classes is because you can't name the class the same thing as the file in the `unittest` framework.

I personally hate the mixed-case naming convention, which is why I opted for this scheme. I have done this several times with already-merged test classes, so I figured it was okay.

I am willing to change this to whatever everybody finds most appropriate.

Reply:
It is fine. I only brought it up since I have seen different schemes used in the codebase. Likely an indicator that we need more convention, but certainly not an urgent matter right now.

Please carry on.